src/modulus_convert.rs

```diff
@@ -0,0 +1,548 @@
+pub mod modulus_conversion {
+
+    use rand::Rng;
+    use rand::RngCore;
+
+    pub struct ReplicatedBinarySecretSharing {
```

Comment:
Would it make more sense to define this as:

```rust
pub struct ReplicatedBinarySecretShare((pub u8, pub u8));
```

Then you access each with `v.0` or `v.1`, which is equally obtuse, but more compact.

Reply:
is it intentionally a tuple of a tuple? aka the double parenthesis `((pub u8, pub u8))`. access would look like:

```rust
let share = ReplicatedBinarySecretShare((0, 1));
let first = share.0.0;
let second = share.0.1;
```